app/views/business_mailer/new_developer_profile.text.erb

```diff
@@ -0,0 +1,15 @@
+New developers at Railsdevs.com!
+
+<%= @business.name %>,
+
+Here's your digest of new developers available for hire at Railsdevs.com:
+
+<%= "-" * 20 %>
+
+<% @developers.each do |dev| %>
+<%= dev.name %>: <%= dev.hero %>
+
+<%= "-" * 20 %>
+
+<% end %>
+Unsubscribe from these emails by editing your preferences in your business profile.
```

Comment:
```suggestion
Unsubscribe from these emails by editing your preferences in your business profile.
<%= edit_business_url(@business) %>
```

Reply:
You can't hyperlink in plain text emails. That will just render out HTML that does nothing.

Reply:
I mentioned this in my comment, but outputting the URL still has value because someone can copy/paste it into their browser.